src/components/BaseNavbar.vue

```diff
@@ -6,7 +6,7 @@
           <img
             class=" h-7"
             src="https://www.binarcode.com/img/brand/logo-dark.png"
```

Comment:
It's best to always have all the assets locally in the project.
External urls may fail due to several reasons (e.g logo name changes on the website).
Download the external images and place them in the `public` directory instead